faucet/main.go

```diff
@@ -389,6 +389,31 @@ func prepareFundMiddleware(client client.Client, fundLimit std.Coins) faucet.Mid
 				return
 			}
 
+			token := r.Context().Value("token").(string)
+
+			keys, _, err := redisClient.SScan(r.Context(), "TOKEN_ADDRESSES", 0, "*:"+request.To, 1).Result()
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+			// DO WE REALLY WANT AN ADDRESS TO NO BE ABLE TO ASK A SECOND TIME MONEY FROM THE FAUCET ?
```

Comment:
I'm for limiting to 1 with this system, but we can give a (way too) huge amount of ugnot, i.e., good for 10k transactions.

Then, if someone hits the limit, we can manually send them additional tokens from an admin account.

WDYT?

Reply:
Yes, works fine :)